Resources/Locale/en-US/logic-gates/logic-gates.ftl

```diff
@@ -0,0 +1,15 @@
+# Examining
+logic-gate-examine-Or = It is currently an OR gate.
+logic-gate-examine-And = It is currently an AND gate.
+logic-gate-examine-Xor = It is currently an XOR gate.
+logic-gate-examine-Nor = It is currently a NOR gate.
+logic-gate-examine-Nand = It is currently a NAND gate.
+logic-gate-examine-Xnor = It is currently an XNOR gate.
+
+# Cycling
+logic-gate-cycle-Or = Switched to an OR gate
+logic-gate-cycle-And = Switched to an AND gate
+logic-gate-cycle-Xor = Switched to an XOR gate
+logic-gate-cycle-Nor = Switched to a NOR gate
+logic-gate-cycle-Nand = Switched to a NAND gate
+logic-gate-cycle-Xnor = Switched to an XNOR gate
```

Comment:
i imagine these can be squished down to just `It is currently {INDEFINITE($gate)} {$gate} gate.` and so on and just pass the uppercased enum name as gate

Reply:
oh didnt know that existed